exercises/practice/two-fer/.approaches/introduction.md

```diff
@@ -0,0 +1,33 @@
+# Introduction
+
+## Default Paramater
+
+The key to this exercise is the optional paramater in the method definition.
+Using a default value there, rather than a conditional, makes the code much more readable, simple, and idiomatic.
+This is a pretty standard solution.
+
+```ruby
+module TwoFer
+  def self.two_fer(name = 'you')
+    "One for #{name}, one for me."
+  end
+end
+```
+
+## Method definition
+
+You can use `class` instead of `module` and use metaclasses instead of `self`, `module_function`, or extend `self`.
+
+## String method
+
+Solutions can also use string templates with `Kernel#sprintf`, `Kernel#format` or `String#%`
```

Comment:
```suggestion
Solutions may also use string templates with `Kernel#sprintf`, `Kernel#format` or `String#%`
```

Reply:
I honestly don't think that changing from `can` to `may` adds anything here. Seeing as this is under the subheading "## String method" it sounds like these things are all referring to different ways of doing string formatting, rather than different ways of solving the exercise. Am I mis-reading this?

Reply:
I think it takes away the implied ability connotation, and gives permission and possibility.  The word "may" gives both permission and possibility, while "can" implies ability (and until they do so, we really do not know if they can or can not).

I am positive that "may" is better for this, because we are suggesting and encouraging the possibility to use these, rather than implying only the ability to do so.